elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -26,9 +26,39 @@ class PfeAccordion extends PFElement {
     return "pfe-accordion.json";
   }
 
-  static get cascadingAttributes() {
+  static get properties() {
     return {
-      on: "pfe-accordion-header, pfe-accordion-panel"
+      color: {
+        title: "Color",
+        type: String
+      },
+      pfeColor: {
+        // @TODO: Deprecate property
+        type: String,
+        prefix: false,
+        alias: "color"
+      },
+      disclosure: {
+        title: "Disclosure",
+        type: String,
+        enum: ["true", "false"],
+        cascade: ["pfe-accordion-header", "pfe-accordion-panel"],
+        observer: "_disclosureChanged"
+      },
+      pfeDisclosure: {
+        // @TODO: Deprecate property
+        type: String,
+        prefix: false,
+        alias: "disclosure"
+      },
+      on: {
```

Comment:
Yes, please don't include this in the accordion definitions since it will be included in the base class' properties.